src/bench/bench.cpp

```diff
@@ -41,18 +42,47 @@ void GenerateTemplateResults(const std::vector<ankerl::nanobench::Result>& bench
 
 } // namespace
 
-benchmark::BenchRunner::BenchmarkMap& benchmark::BenchRunner::benchmarks()
+namespace benchmark {
+
+std::map<std::string, uint8_t> map_priority_level = {
+    std::make_pair("high", PriorityLevel::HIGH),
+    std::make_pair("medium", PriorityLevel::MEDIUM),
+    std::make_pair("low", PriorityLevel::LOW),
+    std::make_pair("all", 0xff)
+};
+
+struct CompPriorities {
+    template <typename T>
+    bool operator()(const T& l, const T& r) const {
+        return l.second < r.second;
+    }
+};
+
+std::string ListPriorities()
```

Comment:
What do you think about returning a sorted `std::vector<std::string>` here to make it more reusable? The `Join()` can easily be applied directly in `bench_bitcoin.cpp`.

Reply:
better to add it when we get an actual use case for it. For now, I think that is fine as is.

Reply:
Okay, I don't agree but not that important so you can mark as resolved.